botocore/utils.py

```diff
@@ -129,6 +129,29 @@
     "streams.dynamodb": "dynamodb-streams",
     "tagging": "resource-groups-tagging-api"
 }
+# These endpoint prefixes represent multiple services. Prior to the service
+# id change, you had to register the endpoint prefix so there was bleed
+# over. We need to preserve that behavior for backwards compatibility.
+DUPLICATE_EVENTS = {
```

Comment:
I think we should be injecting these into the EventAliaser and also add unit tests for this functionality with your own custom duplicate events dict.